core/include/mbox.h

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    core_mbox Mailboxes
```

Comment:
I think the words have different meanings. A "message box" is a box containing messages for public display, like in a faculty hallway, announcing the next X-Mas party. A "mailbox" is the box where incoming mail is put. But I'm not a native speaker...